lametro/templates/partials/board_report_statuses.html

```diff
@@ -0,0 +1,33 @@
+<table class="table">
+    <thead>
+        <tr>
+            <th>Status</th>
+            <th class="desktop-only">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for status, obj in BILL_STATUS_DESCRIPTIONS.items %}
+        <tr>
+            <td>
+                <p class='nowrap'>
+                    {% if obj.search_term != None %}
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008) (the Python style guide) offers this guidance on comparisons to None:

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.

> Also, beware of writing if x when you really mean if x is not None -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

tl;dr -

1. Use `is` or `is not` when comparing to None
2. You really only need to check for `None` explicitly if it means something different than another possible [Falsy](https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/) value, like False or an empty string. 

Here, any Falsy value means we don't have a corresponding search term, so I would recommend simplifying like this:

```suggestion
                    {% if obj.search_term %}
```